tests/devices/unit_tests/p99/test_p99_stage.py

```diff
@@ -26,7 +26,11 @@ async def sim_sampleAngleStage():
 @pytest.fixture
 async def sim_filter_wheel():
     async with DeviceCollector(mock=True):
-        sim_filter_wheel = FilterMotor("p99-MO-TABLE-01:", name="sim_filter_wheel")
+        sim_filter_wheel = FilterMotor(
+            filter_selections=p99StageSelections,
+            prefix="p99-MO-TABLE-01:",
+            name="sim_filter_wheel",
+        )
```

Comment:
```suggestion
        sim_filter_wheel = FilterMotor(
            "p99-MO-TABLE-01:",
            p99StageSelections,
        )
```

nit: prefix first, name comes automatically from DeviceCollector